R/pkg/R/DataFrame.R

```diff
@@ -945,6 +947,19 @@ setMethod("[", signature(x = "DataFrame", i = "missing"),
             select(x, j)
           })
 
+#' @rdname select
+setMethod("[", signature(x = "DataFrame", i = "Column"),
+          function(x, i, j, ...) {
```

Comment:
Are you referring to `...`?
You are probably right... I'm more or less following line 938 above - since this is a i-not-missing overload of the operator/method.
```
 setMethod("[", signature(x = "DataFrame", i = "missing"),
           function(x, i, j, ...) {
```

Perhaps we should take both out? But that might be a breaking change though.